types/foundry/client/pixi/core/interaction/targets.d.ts

```diff
@@ -0,0 +1,32 @@
+export {};
+
+declare global {
+    /**
+     * A subclass of Set which manages the Token ids which the User has targeted.
+     * @extends {Set}
+     * @see User#targets
+     */
+    class UserTargets<TToken extends Token> extends Set<TToken> {
+        constructor(user: User);
+
+        /**
+         * Return the Token IDs which are user targets
+         */
+        get ids(): string[];
+
+        /**
+         * @override
+         */
+        add(token: TToken): this;
```

Comment:
Can you do `override add` instead of the override jsdoc? `@override` is jsdoc nonesense cause javascript doesn't get those toys.

Same with the extends at the top.